hilti/runtime/include/util.h

```diff
@@ -295,10 +337,8 @@ std::string expandEscapes(std::string s);
  * @param escape_quotes if true, also escapes quotes characters
  * @param use_octal use `\NNN` instead of `\XX` (needed for C++)
  * @return escaped string
- *
- * \todo This is getting messy; should use enums instead of booleans.
  */
-std::string escapeBytes(std::string_view s, bool escape_quotes = false, bool use_octal = false);
+std::string escapeBytes(std::string_view s, bitmask<render_style::Bytes> style = render_style::Bytes::Default);
```

Comment:
If swapping to only using positive "escape this" type queries, maybe this default gets changed to some constant that's defined above? Then users can modify from that constant